geomstats/learning/frechet_mean.py

```diff
@@ -355,3 +366,87 @@ def fit(self, X, y=None, weights=None):
         self.estimate_ = mean
 
         return self
+
+    def transform(self, X, y=None, base_point=None):
+        """Lift data to a tangent space.
+
+        Compute the logs of all data point and reshapes them to
+        1d vectors if necessary. By default the logs are taken at the mean
```

Comment:
Is it weird to let have the user be able to pass another base_point, knowing that this class if FrechetMean and passing another base_point now has nothing to do with the mean? They can use the logs directly if they want this, right?

Reply:
I thought it could be useful as one might want to investigate wether using the mean compared to a point (e.g. identity if spd matrices) improves classification.. Otherwise we could put this in a standalone preprocessor or a centralizer transformer... It's not only the log but also the reshaping to 1d that can be useful...

Reply:
I agree, but I'm not sure these should be put as a method of `FrechetMean`. Maybe these should be put as auxiliary functions to preprocess data in the other learning algorithms?

Reply:
The point is any machine learning algorithm from sklearn can be used after this, so I was thinking we should have this preprocessor instead of implementing other learning algorithms that work in a tangent space

Reply:
And it would finally hide all the geometry for ML users. If your point is that this should not be in Frechet mean I don't disagree, but I was thinking it's convenient here and instead of creating yet another class

Reply:
Agreed that a preprocessor would be nice, but we should probably put it in a separate `Projector` class?